continuous-deployment.md

```diff
@@ -0,0 +1,20 @@
+Continuous Deployment
+-----
+
+Continuous Deployment (CD) is a software release process where verified changes to a codebase are deployed immediately and autonomously to a production environment.
+
+You should have continuous deployment set up for your repository, with the following behaviours configured if applicable:
+* Automatically deploying changes to the `main` branch to production
+* Regularly redeploying the application on a schedule
```

Comment:
IIUC scheduled deploys are mostly beneficial on projects using AMIgo images as it ensures the running instances are using recently baked AMIs.

Do we want to qualify this point at all?

```suggestion
* Using an AMIgo AMI and regularly redeploying the application on a schedule
```

Reply:
I agree, that's an important thing to highlight. Updated with your suggestion